src/ZODB/DemoStorage.py

```diff
@@ -309,7 +309,7 @@ def store(self, oid, serial, data, version, transaction):
         if old != serial:
             rdata = self.tryToResolveConflict(oid, old, serial, data)
             self.changes.store(oid, old, rdata, '', transaction)
-            return ResolvedSerial
+            return True
```

Comment:
Why is this store here? (Or the one below?)

Return values from store are no longer used.